### DISCUSSION TITLE
a little confused in setting up b2_env.sh

### DISCUSSION THREAD

**Discussion by noidtw** [erikw/restic-automatic-backup-scheduler#55] [Q&A] [answered]
I've followed the instruction to set up the backup. Just when I run "restic snapshots", it says "Please specify repository location (-r)"

I then follow the instruction on restic  intruction to creat a b2 repository. So, should I put the export repository in 

b2_env.sh.template
RESTIC_REPOSITORY="b2:<b2-repo-name>" ? 

for example RESTIC_REPOSITORY="b2:b01555522e3"
or RESTIC_REPOSITORY="b2:xxbucket:path/to/repo"

Thanks for the excellent work.

**Comment by erikw** [comment, marked as answer]
Hey! I recommend using `make install`. This will copy `b2_env.sh.template` to `/etc/restic/b2_env.sh`.

It is then in `/etc/restic/b2_env.sh` that you should put your RESTIC_REPOSITORY value :)


If you don't use `make install`, you have to copy this file yourself as instructed in https://github.com/erikw/restic-systemd-automatic-backup#2-configure-your-b2-account-locally

Beware, there's a major update to this repo hopefully being merged in the next days which simplifies the instructions. You can already look at it here if you want

https://github.com/erikw/restic-systemd-automatic-backup/pull/45

**Reply by noidtw** [reply]
Yes, i used make to auto setup. However, the b2_env.sh still need manual configuration.

And what i don't understand is the RESTIC_REPOSITORY value that i have to put in it.

Is it the bucket name of my b2?

**Reply by erikw** [reply]
Yes correct you need to edit it

You will get the values for b2_env.sh from step 1

https://github.com/erikw/restic-systemd-automatic-backup#1-create-backblaze-b2-account

when you create your b2 account :)

**Reply by noidtw** [reply]
Thanks, i missed that part. I thought the only thing i need is the b2 account. Sorry, my bad.
